manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "paranext-extension-template",
   "version": "0.0.1",
-  "description": "Extension template for Paranext. Powered by webpack",
```

Comment:
As weird as it sounds, I lean toward leaving `description` here. Changing this line will cause every extension based on this template to have breaking changes when updating, which is going to be a lot of annoying work. I'd suggest adding `displayData` below somewhere like right above `localizedStrings` or something. Basically somewhere to keep it from touching lines that are likely to have changed on many extensions. In hindsight, maybe we shouldn't have put `description` here in the first place, but I think it doesn't hurt to be here and would hurt to have this breaking change on so many templates.

This way, we'll also have a fallback for shortSummary if someone doesn't provide `displayData` for some reason :) (doesn't have to be implemented right now)

@lyonsil do you have an opinion? Maybe this really isn't a useful field to have here, and ripping the bandaid off is better. :thinking:

Reply:
I vote we [rip off the band-aid](https://tenor.com/view/bandaid-rip-take-off-ouch-painful-gif-5085788). 

Let's just get rid of this description and move it to where we are actually going to use the description. It will never get easier to move/remove fields in the manifest than it is now.